neo/app/views/index.scala.html

```diff
@@ -10,12 +10,13 @@
     </div>
 
     <div id='how-to'>
-      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info">How to use this demo</a>
+      <a href="https://www.youtube.com/watch?v=XwlRm0jcsU4" target="_blank" class="btn btn-info btn-sm">How to use this demo</a>
     </div>
-
+      <search-bar></search-bar>
```

Comment:
I wanted to make the icon , title and search bar in single row.Now we have removed that too .I removed that